components/benefit_sponsors/app/controllers/benefit_sponsors/profiles/broker_agencies/broker_agency_profiles_controller.rb

```diff
@@ -26,34 +27,23 @@ class BrokerAgencyProfilesController < ::BenefitSponsors::ApplicationController
           "5" => "employer_profile.plan_years.start_on"
         }.freeze
 
-        def create
-          json = request.body.read
-          body_json = JSON.parse(json)
-          result = ::BenefitSponsors::Services::BrokerRegistrationService.call(body_json["data"], current_user)
-          respond_to do |format|
-            if result.success?
-              format.json { render :status => 201, json: { :message => "Your registration has been submitted. A response will be sent to the email address you provided once your application is reviewed." } }
-            else
-              format.json { render :status => 422, :json => { errors: result.errors.to_h } }
-            end
-          end
-        end
-
         def index
-          authorize self
+          # the _authorize_ method without a BrokerAgencyProfile instance must be called in this format due to the policy not directly corresponding to the controller
+          # calling _authorize_ with no args here will look for a BenefitSponsors::Profiles::BrokerAgencies::BrokerAgencyProfilePolicy instead of a BenefitSponsors::Organizations::BrokerAgencyProfilePolicy
+          authorize [:benefit_sponsors, :organizations, :broker_agency_profile]
```

Comment:
```suggestion
          authorize ::BenefitSponsors::Organizations::BrokerAgencyProfilePolicy, :index?
```

@TristanB17 Is this not working?

Reply:
I believe it's the same thing, but if that's the pattern we want, I can change it

Reply:
Update to use `::BenefitSponsors::Organizations::BrokerAgencyProfilePolicy`